packages/features/settings/layouts/SettingsLayout.tsx

```diff
@@ -105,14 +105,15 @@ const adminRequiredKeys = ["admin"];
 
 const useTabs = () => {
   const session = useSession();
+  const { data: user } = trpc.viewer.me.useQuery();
 
   const isAdmin = session.data?.user.role === UserPermissionRole.ADMIN;
 
   tabs.map((tab) => {
-    if (tab.name === "my_account") {
-      tab.name = session.data?.user?.name || "my_account";
+    if (tab.href === "/settings/my-account") {
```

Comment:
Why are we using href. The name seems to be my_account only.
<img width="477" alt="Screenshot 2023-05-06 at 6 39 40 PM" src="https://user-images.githubusercontent.com/1780212/236626225-10d1bf9c-79b6-46b1-9984-a2c6d8775602.png">